edb/common/ast/base.py

```diff
@@ -243,7 +243,10 @@ def __init__(self, **kwargs):
     def __copy__(self):
         copied = self.__class__()
         for field, value in iter_fields(self, include_meta=False):
-            object.__setattr__(copied, field, value)
+            try:
```

Comment:
Some fields are readonly, and we hit this in the impl of `.replace()`

Pretty scary/ugly to skip here.

Reply:
Add a comment